.circleci/config.yml

```diff
@@ -40,6 +40,32 @@ commands:
       - run:
           name: Running linters and tests
           command: yarn run jest --projects jest.{eslint,stylelint,test}.config.js --maxWorkers=3 --reporters jest-silent-reporter
+  vrt_tests:
```

Comment:
Yes but since it's not clear if we are able to upgrade our github plan by November 13th, it might be that we won't be able to use actions for the time being.

> Legacy plans will not have access to GitHub Actions when it becomes generally available on November 13, 2019

Also, all users part of the commercetools github org have access to circleci. I set it up this afternoon together with @davifantasia, so he's aware of it and knows what to do.

Reply:
OK!